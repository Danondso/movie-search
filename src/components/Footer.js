import React from "react";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles(theme => ({
  footer: {
    marginTop: "73.2vh",
    backgroundColor: "#282c34",
    display: "flex",
    minHeight: "10vh",
    flexDirection: "column",
    color: "white"
  },
  footerTextContainer: {
    marginTop: "4vh"
  }
}));

const Footer = () => {
  const classes = useStyles();
  return (
    <footer className={classes.footer}>
      <div className={classes.footerTextContainer}>
        <span>Created by </span>
        <a
          href="https://github.com/Danondso/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Dublin Anondson
        </a>
      </div>
    </footer>
  );
};

export default Footer;
