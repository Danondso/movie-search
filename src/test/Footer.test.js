import React from 'react';
import Footer from '../components/Footer';
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow } from "enzyme";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

test('Search renders', () => {
    const component = shallow(
        <Footer />
    );
    let tree = toJson(component);
    expect(tree).toMatchSnapshot();
});