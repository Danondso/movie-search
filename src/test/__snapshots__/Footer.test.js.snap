// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <div
    className="makeStyles-footerTextContainer-2"
  >
    <span>
      Created by 
    </span>
    <a
      href="https://github.com/Danondso/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Dublin Anondson
    </a>
  </div>
</footer>
`;
